.reuse/dep5

```diff
@@ -0,0 +1,8 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: Astartectl Test Suite
+Upstream-Contact:  <>
```

Comment:
We used this in other projects (e.g. Edgehog), I'd say to keep it consistent:
```suggestion
Upstream-Contact: SECO Mind Srl
```

Reply:
Solved.